test/e2e/framework/interconnect.go

```diff
@@ -15,11 +15,18 @@
 package framework
 
 import (
+	"time"
+
 	v1alpha1 "github.com/interconnectedcloud/qdr-operator/pkg/apis/interconnectedcloud/v1alpha1"
 
+	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	timeout time.Duration = 10 * time.Second
```

Comment:
One comment from past releases. When we execute tests against cluster deployed remotely (likely at AWS), 10 seconds might not be enough. Maybe we can increase it to 60s.